.orchestra/ci/requirements.txt

```diff
@@ -51,5 +52,6 @@ pytest-parallel
 requests
 requests-toolbelt
 pyyaml
+wget
```

Comment:
As per discussion in the revng PR, I think this can be dropped.

Reply:
Yes. Agree.